tests/utils/server_manager.py

```diff
@@ -33,31 +33,46 @@ def __init__(self, config_paths, client, use_consul=False):
         self._processes = []
 
     def start(self):
-        """Launch the vault server process and wait until its online and initialized."""
+        """Launch the vault server process and wait until its online and ready."""
         if self.use_consul:
             self.start_consul()
 
-        cluster_initialized = False
+        # If a vault server is already running then we won't be able to start another one.
+        # If we can't start our vault server then we don't know what we're testing against.
+        try:
+            self.client.sys.is_initialized()
+        except Exception:
+            pass
```

Comment:
Since this is a test utility, feels prudent to log (i.e., `logger.warn('Vault already initialized')` or what-have-you) when we hit this case so users have as much information as possible no?

Reply:
It outputs the error as part of the exception in the `else` clause.  The [else clause ](https://docs.python.org/2.7/tutorial/errors.html#handling-exceptions) only runs if an exception _wasn't_ thrown by `self.client.sys.is_initialized()` which indicates that Vault is already running.

Reply:
Ah right, checks out! 👍